echopype/consolidate/split_beam_angle.py

```diff
@@ -309,113 +92,173 @@ def _compute_angle_from_complex(
     return theta, phi
 
 
-def get_angle_complex_BB_nopc(
-    ds_beam: xr.Dataset, ed: EchoData
-) -> Tuple[xr.DataArray, xr.DataArray]:
+def get_angle_power_samples(
+    ds_beam: xr.Dataset, angle_params: dict
+) -> Tuple[xr.Dataset, xr.Dataset]:
     """
-    Obtains the split-beam angle data from complex samples from broadband transmit signals
-    without pulse compression.
+    Obtain split-beam angle from CW mode power samples.
 
     Parameters
     ----------
     ds_beam: xr.Dataset
-        An ``EchoData`` beam group containing angle information needed for
-        split-beam angle calculation
-    ed: EchoData
-        An ``EchoData`` object holding the raw data
+        An ``EchoData`` Sonar/Beam_group1 group (complex samples always in Beam_group1)
+    angle_params : dict
+        A dict containing angle_offset/angle_sensitivity params from the calibrated dataset
 
     Returns
     -------
     theta: xr.Dataset
-        The calculated split-beam alongship angle
+        Split-beam alongship angle
     phi: xr.Dataset
-        The calculated split-beam athwartship angle
-    """
+        Split-beam athwartship angle
 
-    # nominal frequency [Hz]
-    freq_nominal = ds_beam["frequency_nominal"]
+    Raises
+    ------
+    NotImplementedError
+        If all ``beam_type`` values are not equal to 1
```

Comment:
Looks like this is not valid anymore? I see `ValueError` in line 148, but not NotImplementedError

Reply:
Good catch! I removed it.